koji/vars/external_repos.yml

```
@@ -80,17 +74,16 @@ external_repos:
     url: file:///mnt/koji/external-repos/www/foreman-rails-1.17-$arch/RPMS.ror51/
   - name: centos-sclo-sclo-rhel-7
     url: http://mirror.centos.org/centos/7/sclo/$arch/sclo/
-  - name: puppetlabs-prod-rhel-6
-    url: file:///mnt/koji/external-repos/www/puppetlabs6-$arch/RPMS.products/
-  - name: puppetlabs-deps-rhel-6
-    url: file:///mnt/koji/external-repos/www/puppetlabs6-$arch/RPMS.dependencies/
-  - name: puppetlabs-pc1-rhel-7
-    url: file:///mnt/koji/external-repos/www/puppetlabs7-$arch/RPMS.pc1/
   - name: qpid-copr
     url: file:///mnt/koji/external-repos/www/qpidcopr-$arch/RPMS.qpid/
   - name: epel7
     url: https://dl.fedoraproject.org/pub/epel/7/$arch/
   - name: flat-ruby27-el8
     url: http://koji.katello.org/kojifiles/releases/split/yum/koji-modules/koji/staged/x86_64/RHEL-8-001-mod/ruby:2.7/
   - name: flat-perl26-el8
-    url: http://koji.katello.org/kojifiles/releases/split/yum/koji-modules/koji/staged/x86_64/RHEL-8-001-mod/perl:5.26/ 
+    url: http://koji.katello.org/kojifiles/releases/split/yum/koji-modules/koji/staged/x86_64/RHEL-8-001-mod/perl:5.26/
+  # Puppet repositories
+  - name: puppetlabs-puppet6-rhel-8
+    url: http://yum.puppetlabs.com/puppet6/el/8/x86_64/
+  - name: puppetlabs-puppet6-rhel-7
+    url: http://yum.puppetlabs.com/puppet6/el/7/x86_64/
```

Oh, I think nowadays puppet.com is preferred. Right now I'm too lazy to find the ticket to prove it.
```suggestion
    url: http://yum.puppet.com/puppet6/el/8/x86_64/
  - name: puppetlabs-puppet6-rhel-7
    url: http://yum.puppet.com/puppet6/el/7/x86_64/
```